agenta-cli/agenta/sdk/decorators/llm_entrypoint.py

```diff
@@ -52,13 +55,36 @@
 
 app.include_router(router, prefix="")
 
-
 from agenta.sdk.utils.debug import debug, DEBUG, SHIFT
 
 
 logging.setLevel("DEBUG")
 
 
+class PathValidator(BaseModel):
+    url: HttpUrl
+
+
+class route(BaseDecorator):
+    # TODO:
+    # - Explain with @route(), for backward compatibility due to @entrypoint limitations.
```

Comment:
Let's also not forget about this. I made some suggestion:

```suggestion
    # This decorator is used to expose specific functions (embedding, retrieval, summarization, etc.)
    # of the LLM workflow as independent endpoints. It is designed for backward compatibility with existing code that uses
    # the @entrypoint decorator, which has certain limitations. By using @route, we can create new
    # routes without altering the main workflow entrypoint. This helps in modularizing the services 
    # and provides flexibility in how we expose different functionalities as APIs.
```

Reply:
On it.